src/Messages/Interactions/TSInfoMessage.m

```diff
@@ -2,16 +2,40 @@
 //  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
 //
 
-#import "NSDate+millisecondTimeStamp.h"
 #import "TSInfoMessage.h"
+#import "NSDate+millisecondTimeStamp.h"
+#import <YapDatabase/YapDatabaseConnection.h>
 
 NS_ASSUME_NONNULL_BEGIN
 
+NSUInteger TSInfoMessageSchemaVersion = 1;
```

Comment:
> It feels like the distinction only exists so as to print a red vs yellow border, which we could achieve much more simply.

Strongly agree!  There's a lot of classes that could be combined.  This would also facilitate our doing something like "make safety number changes switch from red "error" border to orange "info" border after user has visited verification view for that change.